wordpress-mobile/WordPress-iOS
# Muriel colors: change code to new color API

Refs #11683

This changes most calls to the old color API (`WPStyleGuide.wordPressBlue()` to `.primary`, etc). Most changes are strictly according to the conversion table in the spreadsheet.

| Screen recording | Spreadsheet sample |
| ----- | ----- |
| ![color-change-to-new-api-sm](https://user-images.githubusercontent.com/517257/60235812-df76dc00-985d-11e9-8107-4ca64329df11.gif) | ![color table sample](https://user-images.githubusercontent.com/517257/60235426-9e31fc80-985c-11e9-8091-60c300266e7e.png) |

 I've made some changes where it's obvious what an appropriate semantic color would be, such as

```swift
likeButton.setTitleColor(WPStyleGuide.grey(), for: .normal)
// became
likeButton.setTitleColor(.textSubtle, for: .normal)
```

There's also a fun Objective-C API added in `UIColor+MurielColorsObjC.swift`.

To test:
- You can look through the app to see the effects of the color change, but screen-by-screen tweaking is for another PR
- It's key that this makes appropriate changes according to the spreadsheet. So look through the code and its 203 changed files?
- I'm sure you'll notice if something is off

Update release notes:
